### DISCUSSION TITLE
Open box issue

### DISCUSSION THREAD

**Discussion by inspectorbill3** [blackfyre/flsun.community#15] [General]
I purchased an open box flsun qq-s it was missing a few items including sd card. When I plugged the unit in the screen looks like this
![20211004_124443](https://user-images.githubusercontent.com/65734071/135892604-bbe498f1-d0fa-447a-bd87-2c1dea703da9.jpg)

any ideas?

**Comment by blackfyre** [comment]
Hi There!

Welcome to the forum!
You could start with opening up the electronics compartment and check the connections if they are in their proper place and are solid. 
After that you could hook it up to a PC/Laptop/... and get it to communicate with Pronterface of some other gcode terminal to see if the board is intact or not (it might be just the display that's faulty).
If it doesn't connect, it's probably a new board...
